libraries/lwip/configs/lwipopts.h

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
For now lwipcfg and lwipopts headers are probably ok here, but i think a project will pop-up which requires different configuration and will need these 2 files in the project folder to override this default configuration.

I guess we can override this default configuration with -I directives order but we can think about the details later on.